site/docs/how-to/review.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 5
+slug: '/review'
+---
+
+# Peer review infrastructure changes
+
+Eng teams move over from using cloud UIs to IaC because cloud infrastructure changes can lead to outages more often than business logic changes. As a result, it is considered a DevOps best practice to code review infrastructure changes using IaC tools and show the `preview` of the resulting change in pull requests to raise the quality bar and reduce the possibility of mistakes. We do not recommend that cloud infrastructure changes done with IaSQL be code-reviewed using migration systems since the IaSQL schema is defined by [modules](../concepts/module.md) that can be upgraded and changed over time. Furthermore, we recommend complex, or delicate, infrastructure changes should be treated as transactions in IaSQL. [IaSQL transactions](../concepts/transaction.md) are a bit different from normal DB transactions in that they are multiuser and can be previewed with the [`iasql_preview`](../modules/builtin/iasql_functions.md) function for the changes that the ongoing transaction would create in the cloud, similar to Pulumi’s `preview` or Terraform’s `plan`.
+
+We set forth the best convention to review infrastructure changes within transactions which:
+- works with any cloud identity and access management structure
+- works with IaSQL schema upgrades
+- works with any version control system, not just GitHub, without the need for privileged access
+- preserves the ability to `iasql_preview` changes in development and get the type-safety feedback from the dashboard editor
+
+```sql title="Create review for an infrastructure change within a transaction"
+SELECT * FROM iasql_begin();
+
+UPDATE log_group SET log_group_name = 'test' WHERE id = 4;
+
+SELECT * FROM iasql_create_review('My review' )
```

Comment:
```suggestion
SELECT * FROM iasql_create_review('My review')
```

Reply:
should we add in this example the `description` too so users can know how to use it?

Reply:
I agree with @aguillenv that this doc should have the description column filled in, but maybe mention that it's optional?